lockfile/src/csharp.rs

```diff
@@ -95,16 +94,19 @@ pub struct CSProj;
 
 #[derive(Debug, Deserialize, PartialEq, Eq)]
 pub struct PackageReference {
-    #[serde(alias = "Include", default)]
+    #[serde(rename = "@Include", default)]
```

Comment:
Curious why these were changed, considering alias should be a superset here?

Reply:
Reverted, I was following the readme, but it didn't need to be changed except for the adding @ for the attribute.